core/src/test/java/org/apache/calcite/sql/parser/SqlUnParserTest.java

```diff
@@ -25,4 +36,25 @@ class SqlUnParserTest extends SqlParserTest {
     return super.fixture()
         .withTester(new UnparsingTesterImpl());
   }
+
+  /**
+   * Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6977">[CALCITE-6977]
+   * Unparse DELETE SQL throws unsupported exception</a>.
+   */
+  @Test void testSqlDeleteSqlBasicCallToString() {
```

Comment:
actually I have one more request: SqlUnparserTest is an extension of SqlParserTest, which runs all the same tests with a different fixture.
I think this function should be moved there.

Reply:
I'm totally agree with you,  i have moved this function into  SqlParserTest class, thanks all you guys!😊